project/Build.scala

```diff
@@ -16,7 +16,7 @@ object B extends Build {
   override lazy val settings = super.settings ++ Seq(
     organization := "com.typesafe.genjavadoc",
     version := "0.9-SNAPSHOT",
-    scalaVersion := "2.10.4",
+    scalaVersion := "2.11.5",
```

Comment:
I think `master` can be on `2.11` since there is no need for additional commits when cross releasing as `+publishSigned` should do all the work.

Also it can be on latest scala, because `crossScalaVersions` contains all scala versions. I would even change this line to read `crossScalaVersions.value.last` to remove duplication.